### DISCUSSION TITLE
Unnecessary GetDotnetCommand() Messages?

### DISCUSSION THREAD

**Discussion by JooleanLogic** [Chizaruu/com.tsk.ide.vscode#50] [General]
Can I suggest that GetDotnetCommand() in ProjectGeneration.cs on line 1045 only throw the [Win32Exception](https://github.com/Chizaruu/com.tsk.ide.vscode/blob/8b148a68e9d438de9a7ae6ee8c5da2a8419206ea/com.tsk.ide.vscode/Editor/ProjectGeneration/ProjectGeneration.cs#L1101) message if none of the 'possibleCommands' are matched?

I'm on linux and get the "Command '/bin/zsh' not found or not accessible." debug message whenever I start Unity or launch vscode.
As only one of those possibleCommands needs to succeed, it seems unnecessary to log output about which ones failed, unless they all failed.

PS. Apologies if this is wrong place for this. I'm not fluent in github processes and your discord link failed. Delete at will.

**Comment by Chizaruu** [comment]
That's alright; it would be preferable to create the post in the issues tab.

However, thanks for bringing this to my attention. :)

I'll sort it out for you soon!

**Reply by JooleanLogic** [reply]
Cheers mate and thanks for the tip.

**Comment by Chizaruu** [comment]
This would be the discord link. https://discord.gg/WxFG6Wm5hk

**Comment by Chizaruu** [comment]
Closing due to PR #52
